dev_tools/bin/bump-starter-frontend-version

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative '../lib/solidus/bumper'
+
+candidate_version = ARGV[0]
+
+path = File.expand_path('../../core/lib/generators/solidus/install/app_templates/frontend/starter.rb', __dir__)
+from = File.read(path)[/solidusio\/solidus_starter_frontend\/raw\/(.+)\/template.rb/, 1]
```

Comment:
I'm failing to understand how it works :thinking:

Reply:
From the file content, it matches the regular expression and returns what matches the thing inside the parenthesis, in this case `(.+)`, which will be `main` or the branch name (eg. `v3.2`).

Reply:
I did this test, in the sandbox:

```ruby
path = File.expand_path('../core/lib/generators/solidus/install/app_templates/frontend/starter.rb', __dir__)
File.read(path)[/solidusio\/solidus_starter_frontend\/raw\/(.+)\/template.rb/, 1]
# => "main"
```

Reply:
Oh, sorry :facepalm: First review on Monday :sweat:

Reply:
No need to say sorry! It's probably a signal that it's not super intuitive, and I'm open to more readable ways of expressing this if you have any suggestions.

Reply:
Done!